ui/ts/models/log.ts

```diff
@@ -77,13 +76,22 @@ module Models {
         return url;
       }
 
-      constructor() {
+      constructor(nodeId?: string) {
         this.level(Utils.Format.Severity(2));
         this.max(null);
         this.startTime(null);
         this.endTime(null);
         this.pattern(null);
-        this.node(null);
+        this.node(nodeId);
+        // Add the QueryCache after initialization in order to not fetch without
```

Comment:
When the queryCache was up top, it immediately starts a query, which happens during the construction of the object, not after the constructor runs.  So the first query it sends is to a "null" nodeID ("local").  Which then gets refreshed after the constructor is called.  Moving the code here stops that first call from happening prematurely.
I'm working on cleaner solution right now, but doing it this way is perfectly acceptable.